uniffi_bindgen/src/lib.rs

```diff
@@ -151,7 +151,32 @@ fn load_bindings_config<BC: BindingGeneratorConfig>(
     }
 
     // Leverage serde to convert toml::Value into the config type
-    Ok(toml::Value::from(config_map).try_into()?)
+    toml::Value::from(config_map)
+        .try_into()
+        .context("Generating bindings config from toml::Value")
+}
+
+/// Binding generator config with no members
+#[derive(Clone, Debug, Hash, PartialEq, PartialOrd, Ord, Eq)]
+pub struct EmptyBindingGeneratorConfig;
+
+impl BindingGeneratorConfig for EmptyBindingGeneratorConfig {
+    fn get_entry_from_bindings_table(_bindings: &toml::Value) -> Option<toml::Value> {
+        None
+    }
+
+    fn get_config_defaults(_ci: &ComponentInterface) -> Vec<(String, toml::Value)> {
+        Vec::new()
+    }
+}
+
+impl<'de> Deserialize<'de> for EmptyBindingGeneratorConfig {
```

Comment:
Can we replace this with a derive on the `EmptyBindingGeneratorConfig`? or does serde complain?

Reply:
It's a unit struct, but we want to deserialize from an empty map, so the `derive(Deserialize)` doesn't work.  I'll add a comment about this.